packages/app/design-system/pressable.tsx

```diff
@@ -1,12 +1,15 @@
 import { ComponentProps } from 'react'
-import { Pressable as DripsyPressable } from 'dripsy'
+import { Pressable as DripsyPressable, useSx } from 'dripsy'
 
 import { tw as tailwind } from 'app/design-system/tailwind'
 
-type TextProps = { tw?: string } & ComponentProps<typeof DripsyPressable>
+type TextProps = { className?: string } & ComponentProps<typeof DripsyPressable>
```

Comment:
why do you rename tw to className?
Is this a design decision?

Reply:
Because code completion wasn't working for tw (WebStorm)